contracts/RollupPoB.sol

```diff
@@ -79,122 +84,175 @@ contract RollupPoB is RollupPoBHelpers{
 
     /**
      * @dev RollupPoB constructor
      * Set first block where the first slot begin
      * @param _rollup rollup main smart contract address
      * @param _maxTx maximum transactions
      */
-    constructor(address _rollup, uint256 _maxTx, address payable burnAddress) public {
+    constructor(address _rollup, uint256 _maxTx, address payable burnAddress, address payable opDefaultAddr, string memory urlDefault) public {
         require(_rollup != address(0), 'Address 0 inserted');
         rollupInterface = RollupInterface(_rollup);
         genesisBlock = getBlockNumber() + DELAY_GENESIS;
         MAX_TX = _maxTx;
         burn = burnAddress;
+        opDefault = Operator(opDefaultAddr, opDefaultAddr, opDefaultAddr, opDefaultAddr, urlDefault);
     }
 
     /**
      * @dev save the winning operator and return the amount to the previous winner
      * @param slot block number
-     * @param forgerAddress address to forge
-     * @param beneficiaryAddress address to receive fees
-     * @param withdrawAddress address to withdraw amount
-     * @param bonusAddress address to receive bonus in SC
+     * @param op operator address and url
      * @param useBonus to use the saved bonus
+     * @param value bid value
      */
     function doBid(
         uint32 slot,
-        address payable beneficiaryAddress,
-        address forgerAddress,
-        address withdrawAddress,
-        address bonusAddress,
-        bool useBonus
-    ) internal {
-        uint256 amount = msg.value;
+        Operator memory op,
+        bool useBonus,
+        uint256 value
+    ) internal returns (uint) {
+        uint256 amount = value;
+        uint burnAmount = 0;
         if (useBonus) {
-            require(msg.sender == bonusAddress, "To use bonus it is necessary that sender is the bonusAddress");
-            amount += bonusBalance[bonusAddress];
-            bonusBalance[bonusAddress] = 0;
+            require(msg.sender == op.bonusAddress, "To use bonus it is necessary that sender is the bonusAddress");
+            amount += bonusBalance[op.bonusAddress];
+            bonusBalance[op.bonusAddress] = 0;
         }
         if(slotBid[slot].initialized) {
             uint minNextBid = slotBid[slot].amount + (slotBid[slot].amount * percentNextBid)/100;
             require(amount >= minNextBid, 'Ether send not enough to outbid current bid');
             uint bonus = (slotBid[slot].amount * percentBonus)/100;
             _returnBid(slotBid[slot].amount, bonus, slotWinner[slot]);
             infoSlot[slot].accumulatedBonus += bonus;
-            burn.transfer(amount - slotBid[slot].amount - bonus);
+            burnAmount = amount - slotBid[slot].amount - bonus;
         } else {
             require(amount >= MIN_BID, 'Ether send not enough to enter auction');
-            burn.transfer(amount);
+            opDefault.beneficiaryAddress.transfer(amount);
```

Comment:
`amount` includes `msg.value` and `bonusBalance` if the `msg.sender` choose to use it.
Since it will be the first bid for a given slot, all the `amount` should go to the default operator regardless its source, either `msg.value` or `bonusBalance`